### PR TITLE
[WIP] 360Giving Data Quality Tool Sprint 2019-02-20 -> 2019-03-12

### PR DESCRIPTION
**Pull request by Bjwebb**
Dev deploy for this temporary master branch - http://master-360-data-quality-tool.cove-360-dev.default.threesixtygiving.uk0.bigv.io/

Still TODO:
- [x] Fix the additional checks explanation page
- [x] Update the secret data tests
- [x] Release lib-cove https://github.com/OpenDataServices/lib-cove/pull/11 and lib-cove-web https://github.com/OpenDataServices/lib-cove-web/pull/13 and update the requirements in this PR to use the released version
- [x] Update lib-cove-ocds to a released version

### CODE REVIEW

**Comment by BibianaC** [comment]
Fix additional checks can be done outside the sprint work so I will mark it as done.

**Comment by Bjwebb** [comment]
Before merging this, we should update the lib-cove-ocds requirement to a released version (which I think will have to be a new release).

**Comment by odscjames** [comment]
Might help this branch - lib-cove has just been updated to the latest version in the master branch.

File: setup.cfg [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [flake8]
 exclude=build,.git,.ve*,.hypothesis
-ignore=E123,E128,E265,E501,W601,W293
+# extend-ignore uses flake8's default ignore list, and adds to it
+extend-ignore=E128,E265,E501,W601,W293,W605
```
Comment (odscjames) [inline setup.cfg]: Can we document why we are adding exceptions in https://opendataservices.gitbook.io/developer-docs/code/code-python#coding-style ? That will help share knowledge round the team. Thanks :-)

File: cove_360/templates/cove_360/explore.html [hunk 2]
```diff
@@ -143,71 +160,208 @@ <h3 class="panel-title panel-title-explore">
...
+            {% endif %}
+          {% endwith %}
+          {% with validation_errors=validation_errors_grouped.other error_prefix='other-' %}
+            {% if validation_errors %}
+              <div class="panel panel-default">
+                <div class="panel-heading">
+                  <h4>Other</h4>
+                </div>
+                <div>
+                  <p>Other validation errrors.</p>
```
Comment (odscjames) [inline cove_360/templates/cove_360/explore.html]: Typo - err ***r*** ors

**Pull request merged**